src/native/popup/popup.js

```diff
@@ -36,6 +43,10 @@ type AndroidApp = {|
     version? : string
 |};
 
+function isAlreadyAuthorizedUser() : boolean {
+    return window.document.cookie && window.document.cookie.length ? AUTH_COOKIES.some(cookie => window.document.cookie.includes(cookie)) : false;
```

Comment:
Can we use `String.prototype.indexOf` instead of `String.prototype.includes`? IE 11 does not support string includes() and I don't think we have a polyfill for it.
```suggestion
    return window.document.cookie && window.document.cookie.length ? AUTH_COOKIES.some(cookie => window.document.cookie.indexOf(cookie) !== -1) : false;
```

Reply:
Bummer.  No polypill.  `indexOf` it is.